Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -629,6 +634,8 @@ void Wiimote::Update()
 	if (Step())
 		return;
 
+	static ControlState s_sideways_toggle_last = 0.0, s_upright_toggle_last = 0.0;
```

Comment:
Please don't use function-scoped static variables. 

Make them private members in the Wiimote class instead.